common/src/main/java/com/commercetools/payment/utils/impl/PaymentConnectorHelperImpl.java

```diff
@@ -20,7 +20,6 @@
     public CompletionStage<HttpRequestResult> sendHttpGetRequest(String url) {
         HttpRequest request = HttpRequest.of(HttpMethod.GET, url);
 
-        // TODO: must be injected, instead of initialization!!!
         HttpClient client = SphereClientFactory.of().createHttpClient();
 
```

Comment:
remove unused empty lines

Reply:
there are no unused empty lines in this method. Just declarations are separated by one empty line for better readability.